### DISCUSSION TITLE
Changing the message on boot

### DISCUSSION THREAD

**Discussion by Lyall-A** [mriscoc/Ender3V2S1#1361] [General]
I'm trying to find out how to change the message that shows on the LCD during boot (the stuff that says MRiscoC etc etc custom build etc etc) but I can't see it anywhere apart from the .a binaries in Marlin/libs/proui, is there a way to change the text?

**Comment by Lyall-A** [comment]
Look's like its under libproui.a, is the source for this public? I want to change/remove the popup

**Comment by mriscoc** [comment]
For remove the popup disable the line:

https://github.com/mriscoc/Ender3V2S1/blob/78c615a70db43cdc17254ead569b19af357c3c3c/Marlin/Configuration.h#L3413

Alternatively, you can use the official Marlin repository (https://github.com/MarlinFirmware/Marlin) with the Marlin ProUI.

**Comment by Lyall-A** [comment]
This is what I wanted! Thanks
